modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -299,7 +314,7 @@ public ObjectDefinition getObjectDefinition(
 		long companyId, String restContextPath) {
 
 		Map<Long, ObjectDefinition> objectDefinitions =
-			_objectDefinitionsMap.get(restContextPath);
+			_objectDefinitionsMap.get(restContextPath + companyId);
```

Comment:
Reuse _objectDefinitionInstanceKey here?

Reply:
Mmmm, I wasn't sure about that. As we are accessing to the `_objectDefinitionMap` from the outside, maybe there is or would be a case where we want to get an ObjectDefinition that is not the actual one. That's why I prefer to not use the `_objectDefinitionInstanceKey`
If you consider that it is too dangerous to use and it is better to use the global variable, I can change it.

Reply:
Makes sense.